src/main.ts

```diff
@@ -313,8 +315,16 @@ ipcMain.handle('browser:start', async (event, url?: string) => {
 
 ipcMain.on('browser:stop', async () => {
   console.info('browser:stop event received')
+
   if (currentBrowserProcess) {
-    await currentBrowserProcess.close()
+    // macOS & windows
+    if ('close' in currentBrowserProcess) {
+      await currentBrowserProcess.close()
+      // linux
+    } else {
+      currentBrowserProcess.kill()
+    }
+
```

Comment:
Probably not the best of the best approach here but considering I want to raise removing the puppeteer dependency it seems good enough.

Basically we just check if we have the `close` from the puppeteer object and if not we just call `kill`.